woocommerce/woocommerce-gutenberg-products-block
# Move withComponentId to hocs directory

In #791 and #779 we are creating the HOCs under a `hocs` directory. This PR moves the existing HOC `withComponentId` to that directory as well.

### How to test the changes in this Pull Request:

1. Create a post and add a _Product Categories List_.
2. Verify there are no JS errors neither in the editor nor the frontend.

### Changelog

> Internal: moved `withComponentId` HOC under `hocs` directory.
